llvm/docs/GettingInvolved.rst

```diff
@@ -331,6 +331,11 @@ don't find anyone present, chances are they happen to be off that day.
     - Monthly, 4th Wednesday of the month at 9:30am PT, for 30 minutes.
     - `Google meet <https://meet.google.com/pdd-dibg-cwv>`__
     - English
+  * - Rotating hosts
+    - Getting Started, beginner questions, new contributors.
+    - Every Tuesday at 2 PM ET (11 AM PT), for 30 minutes.

```

Comment:
Suggestion for a follow-up (not insisting on it, either!), convert all of the entries to specify their time in UTC but use a link to something like https://www.timeanddate.com/worldclock/converter.html?iso=20240201T180000&p1=136&p2=179&p3=224&p4=248 so that people have a better idea of when that is. It's not perfect thanks to stuff like daylight savings time changing throughout the year, but maybe there's a feature on that site (or one of the other ones) to account for that.

Reply:
oh! great idea!

Reply:
Unfortunately we do have a scheduling conflict due to daylight savings if the meeting moves from 2pm local time to 3pm local time in March for DST. Since (I think) all the current hosts have DST start and end on the same date, hopefully having the calendar event set based on ET is clear, and then when attendees view the LLVM calendar on the web (https://calendar.google.com/calendar/u/0/embed?src=calendar@llvm.org) it shows the corresponding UTC time accurately all year. If they add the LLVM  calendar to their own calendar-ing app of choice (this is what I do), it should show all the meetings in their local time. Ack that none of the solutions are perfect because *handwaves*.

Likewise, I'll pass on the follow up only for those same reasons. I don't want to cause this document to conflict with the actual timezones people used for their calendar events.

Reply:
Hm, I notice there is confusion on the #beginners discord channel today about this exact problem. I will try to follow up with some documentation on this page for how to find these events in your local timezone.

Reply:
It's fine for that work to be a follow-up after landing these changes, IMO.